atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -317,6 +318,7 @@ private enum State {
     private final KeyValueSnapshotMetricRecorder snapshotEventRecorder;
     private final ReadSentinelHandler readSentinelHandler;
     private final KeyValueSnapshotReader keyValueSnapshotReader;
+    protected final KeyValueSnapshotReaderManager keyValueSnapshotReaderManager;
```

Comment:
might have missed this: is this needed by the SerializableTransaction directly?

Reply:
yes, it's used by SerializableTransaction to construct new ReadOnlyTransaction instances

Reply:
ah, got it